distributions/jamulus-server.service

```diff
@@ -8,7 +8,7 @@ Type=simple
 Restart=always
 RestartSec=1
 User=jamulus
-ExecStart=/bin/sh -c '/usr/bin/jamulus -s -n -l /var/log/jamulus -e jamulus.fischvolk.de -g -o "$(uname -n);;"'
+ExecStart=/bin/sh -c 'exec /usr/bin/jamulus -s -n -l /var/log/jamulus -e anygenre3.jamulus.io -g -o "$(uname -n);;"'
```

Comment:
The commit message makes it sound like this would be a cosmetical change from fischvolk.de to jamulus.io while in reality it does change the directory server. Is the change to anygenre**3** intentional? I guess it's ok as this will most likely only affect users who set up new servers.

(Same applies to raspijamulus.sh)

Reply:
I wondered about which directory server to use for the example. I chose 3 because 1 and 2 spend most of their time full already.

Reply:
Yes, this sounds good to me, I just wanted to ensure that it was intentional.